client/my-sites/plugins/disconnect-jetpack/disconnect-jetpack-dialog.jsx

```diff
@@ -1,83 +1,95 @@
 /**
  * External dependencies
  */
-var page = require( 'page' ),
-	React = require( 'react' );
+import page from 'page';
+import React, { Component } from 'react';
+import { translate } from 'i18n-calypso';
```

Comment:
Let's wrap the class in the higher order component [localize](https://github.com/Automattic/i18n-calypso#localize). We may miss some change events otherwise.